rest/importtest/import_test.go

```diff
@@ -1917,7 +1917,7 @@ func TestImportRevisionCopyUnavailable(t *testing.T) {
 	assert.Equal(t, 404, response.Code)
 }
 
-func TestImportComputeStatOnDemand(t *testing.T) {
+func TestImportComputeStatOnDemandGet(t *testing.T) {
 	base.SkipImportTestsIfNotEnabled(t)
```

Comment:
is this skip required here, especially since you set `AutoImport: false`?

Also, if you set `AutoImport: true` does this do double counting anywhere? That's the thing I'm most worried about in this PR.